webview-ui/src/InspektorGadget/GadgetSelector.tsx

```diff
@@ -4,11 +4,11 @@ import { GadgetCategory } from "./helpers/gadgets/types";
 import { configuredGadgetResources } from "./helpers/gadgets";
 
 export interface GadgetSelectorProps {
-    category: GadgetCategory
-    id: string
-    className: React.HTMLAttributes<any>['className']
-    required?: boolean
-    onResourceChanged: (resource: string | null) => void
+    category: GadgetCategory;
+    id: string;
+    className?: string;
```

Comment:
The old type `React.HTMLAttributes<any>['className']` evaluates to `string | undefined`, so the simpler type is equivalent. (I guess the complex type name could be used by some tooling for intellisense but I don't think it provides any value for us.)